Gemfile.lock

```diff
@@ -211,21 +211,21 @@ GEM
       thor (>= 0.14, < 2.0)
     json (2.3.0)
     jsonapi-renderer (0.2.2)
-    kaminari (1.1.1)
+    kaminari (1.2.0)
       activesupport (>= 4.1.0)
-      kaminari-actionview (= 1.1.1)
-      kaminari-activerecord (= 1.1.1)
-      kaminari-core (= 1.1.1)
-    kaminari-actionview (1.1.1)
+      kaminari-actionview (= 1.2.0)
+      kaminari-activerecord (= 1.2.0)
+      kaminari-core (= 1.2.0)
+    kaminari-actionview (1.2.0)
       actionview
-      kaminari-core (= 1.1.1)
-    kaminari-activerecord (1.1.1)
+      kaminari-core (= 1.2.0)
+    kaminari-activerecord (1.2.0)
       activerecord
-      kaminari-core (= 1.1.1)
-    kaminari-core (1.1.1)
+      kaminari-core (= 1.2.0)
+    kaminari-core (1.2.0)
```

Comment:
### [kaminari-core](https://github.com/kaminari/kaminari)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.1.1 → 1.2.0

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **101** commits. See the full changes on [the compare page](https://github.com/kaminari/kaminari/compare/v1.1.1...v1.2.0).

These are the first 10 commits:
- (f24857a) [Remove magic comment](https://github.com/kaminari/kaminari/commit/f24857a7723b386bbbef75cd9ddb7a8f2e4c228e)
- (f7da7cc) [Merge pull request #922 from swamp09/remove_magic_comment](https://github.com/kaminari/kaminari/commit/f7da7cc9f3144a9764c89eccbe4eda555d48c0b5)
- (0ff8c30) [Remove the rest of the magic comments for file encoding](https://github.com/kaminari/kaminari/commit/0ff8c3094d720dfeb683f9c398a7d9c50ca4b04f)
- (8f238ac) [kaminari-core and kaminari-activerecord updated 1.1.1 .](https://github.com/kaminari/kaminari/commit/8f238ace178ece4dcd030f7dd4f0d3a569c17f2a)
- (b061a06) [Remove version from 'gem "kaminari-core"' and 'gem "kaminari…](https://github.com/kaminari/kaminari/commit/b061a068614c5298a4143b973706f2abac1e70c0)
- (4727eda) [Merge pull request #923 from emj-naka/remain_old_version_101](https://github.com/kaminari/kaminari/commit/4727eda1994eb271062f72ed863be1ec498b441c)
- (59a550e) [Fix total_count on loaded relation](https://github.com/kaminari/kaminari/commit/59a550e932d30ebf0c1aa027f4f51fa776503c33)
- (cd85dd6) [Add PR categories we don't generally accept to CONTRIBUTING.…](https://github.com/kaminari/kaminari/commit/cd85dd670a73bc88e047b158fb0faca0531ed867)
- (9976b6e) [Update test description [ci skip]](https://github.com/kaminari/kaminari/commit/9976b6ee172e12bbe2a5cc1b0f4f87d83343b754)
- (d38c684) [Revert "Deactivesupport"](https://github.com/kaminari/kaminari/commit/d38c6849e02bb57289a3c6e7db062d9746d0aa4c)


</details>